services/app/src/Constants.tsx

```diff
@@ -192,6 +192,12 @@ export const INIT_DELAY = {
   SILENT_LOGIN_MILLIS: 1000,
 };
 
+export const CONTENT_SET_FULL_NAMES: {[k: string]: string} = {
```

Comment:
Since you're introducing this as a new constant (thank you), could you do a quick find and replace across the code?

Reply:
done.